### ISSUE TITLE
Why is this broken?

### ISSUE THREAD

**Issue by resistorsoftware** [TryGhost/Starter#42]
To install and use Ghost on MacOS, it is tuned (on my system anyway) to cry foul unless the install sees Node 14.15, so that is what I am running. But this Starter Theme install, it is looking for Node 10? or 12? What am I supposed to do with this? 

I want to create my own theme using the starter, but apparently, I need to backtrack to the old Node? Is that current?

yarn
yarn install v1.22.10
[1/5] 🔍  Validating package.json...
warning ghost-starter-theme@1.0.0: The engine "ghost" appears to be invalid.
warning ghost-starter-theme@1.0.0: The engine "ghost-api" appears to be invalid.
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error gscan@3.5.4: The engine "node" is incompatible with this module. Expected version "^10.13.0 || ^12.10.0". Got "14.15.3"
error Found incompatible module.

**Comment by resistorsoftware** [comment]
I found this to be the solution. One, I tried out using the **nvm** utility. It is primitive but allowed me to install node 12.xx. So I used that to install the starter template and get the yarn command to succeed. 

There was trouble ahead though, as the original install was done with node 14.15 (listed as working and stable for Ghost), so my overall install failed due to mixed compilation with Node. So I was forced to re-do everything with Node 12.xx series. 

Then I added TailwindCSS but there was trouble with this as the latest works with PostCSS 8, but this Ghost system and Gulp are only good for PostCSS 7. So then I backed to use PostCSS 7 with the Tailwind bridge to old PostCSS 7, and that failed due to a carat in the packaging. In other words, installing @PostCSS^7 failed but @PostCSS7 worked. 

It took me ~ 3 hours to thus deploy a Ghost starter template with the latest TailwindCSS, and I have no idea if it actually works at all, as I have not tried editing the styles, purging the unused classes, or publishing to the cloud. 

My experience is unique, and I am sure the community at large has no such issues like mine, so I take my problems with a grain of salt, and assume this project will live for many years into the future, getting better at each iteration.
